lib/utils.js

```diff
@@ -134,8 +134,9 @@ function parseStack(err, cb) {
             lineno: line.getLineNumber(),
             'function': getFunction(line),
         }, isInternal = line.isNative() ||
-                        (frame.filename[0] !== '/' &&
-                         frame.filename[0] !== '.');
+                        (frame.filename[0] !== path.sep &&
+                         frame.filename[0] !== '.' &&
+                         frame.filename.indexOf(':\\') != 1);
```

Comment:
Yes, it's just one backslash but it has to be escaped with another one inside the string.  I am now changing the inequality check.